test/Microsoft.SqlTools.ServiceLayer.UnitTests/EditData/CellUpdateTests.cs

```diff
@@ -159,6 +159,17 @@ public void BoolInvalidFormatTest()
         }
 
         [Theory]
+        [InlineData("24:00:00")]
+        [InlineData("105:00:00")]
+        public void TimeSpanTooLargeTest(string value)
```

Comment:
If you're feeling really ambitious you could make a test that verifies that timespans within the allowed range don't throw.